mix.exs

```diff
@@ -81,7 +81,7 @@ defmodule Elixium.Mixfile do
         # 8 Megabyte block size
         block_size_limit: 8_388_608,
 
-        unix_key_address: "~/.keys",
+        unix_key_address: "/keys",
```

Comment:
I dont think unix_key_address is used anywhere anymore, you can safely remove it

Reply:
Awesome!